frontend/src/components/OrderPage.js

```diff
@@ -113,6 +122,44 @@ class OrderPage extends Component {
     this.getOrderDetails(this.state.orderId);
   }
 
+  handleWebln = (data) => {
+    getWebln()
+      .then((webln) => {
+        // If Webln implements locked payments compatibility, this logic won't be necesary anymore
+        if (data.is_maker & data.status == 0) {
+          webln.sendPayment(data.bond_invoice);
+          this.setState({ waitingWebln: true, openWeblnDialog: true});
+        } else if (data.is_taker & data.status == 3) {
+          webln.sendPayment(data.bond_invoice);
+          this.setState({ waitingWebln: true, openWeblnDialog: true});
+        } else if (data.is_seller & (data.status == 6 || data.status == 7 )) {
+          webln.sendPayment(data.escrow_invoice);
+          this.setState({ waitingWebln: true, openWeblnDialog: true});
+        } else if (data.is_buyer & (data.status == 6 || data.status == 8 )) {
+          webln.makeInvoice(data.amount).then((invoice) => {
+            this.sendWeblnInvoice(invoice)
```

Comment:
I think a `catch` will do the trick, but not sure

Reply:
I assume you cannot test it since this feature can only be tested on mainnet. While it is a bit of a hassle, it possible to test the submission of the invoice payout on the mainnet platform by taking your own order with another Robot. After the escrow is locked (20K Sats), you can collaboratively cancel to unlock every hodl invoice.